guides/creating-a-custom-form-field-control.md

```diff
@@ -339,15 +339,30 @@ controlType = 'example-tel-input';
 
 #### `setDescribedByIds(ids: string[])`
 
-This method is used by the `<mat-form-field>` to specify the IDs that should be used for the
-`aria-describedby` attribute of your component. The method has one parameter, the list of IDs, we
-just need to apply the given IDs to our host element.
+This method is used by the `<mat-form-field>` to set element ids that should be used for the
```

Comment:
Nice, I think this makes it a lot more clear why both properties exist / what they're for